app/views/govuk_publishing_components/components/docs/reorderable_list.yml

```
@@ -0,0 +1,92 @@
+name: Reorderable list
+description: A list of items that can be reordered
+body: |
+  List items can be reordered by drag and drop or by using the up/down buttons.
+  On small viewports the drag and drop feature is disabled to prevent being triggered
+  when scrolling on touch devices.
+
+  This component uses SortableJS - a JavaScript library for drag and drop interactions.
+  When JavaScript is disabled a set of inputs will be shown allowing users to provide
+  an order index for each item.
+
+  When this component is embedded into a form and that form is submit you will receive a
+  parameter of ordering (which can be customised with the input_name option).
+  This is submitted as a collection which Rails will interpret as hash of item id => ordering position.
+  
+  For example, for two items with id "a", "b" that are ordered accordingly, you'd receive a submission
+  of `ordering[a]=1&ordering[b]=2` and rails will translate that into params as:
+  
+  ```
+    {
+      "ordering" => {
+        "a" => "1",
+        "b" => "2"
+      }
+    }
+  ```
```

```suggestion
  When this component is embedded into a form and that form is submit you will receive a
  parameter of `ordering` (which can be customised with the `input_name` option).
  This will contain item ids and ordering positions in a hash.

  For example, for two items with id "a" and "b" that are ordered accordingly,
  you'd receive a submission of `ordering[a]=1&ordering[b]=2`, which Rails can
  translate to `"ordering" => { "a" => "1", "b" => "2" }`.
```

A few minor modifications to this.

I noticed that using  \`\`\` doesn't work properly in these docs too (pops out inline) Since the code is simple we should probably just put on a single line rather than using an alternative syntax to get a pre (for example `~~~` works).